app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+import re
+
+
+def move_file(command: str) -> None:
+    try:
+        command, source_path, destination_path = command.split()
+    except ValueError:
+        print("Wrong command format")
+    else:
+        if command == "mv" and source_path != destination_path:
+            if not os.path.isfile(source_path):
+                raise ValueError(f"File '{source_path}' doesn't exist")
+            source_path = os.path.join(*re.split(r"[\\/]", source_path))
+            destination_path = os.path.join(
+                *re.split(r"[\\/]", destination_path)
+            )
+            if (
+                    source_path.endswith(os.sep)
+                    or destination_path.endswith(os.sep)
```

Comment:
this block of code is unnecessary if you use try-except when you work with file.
also  `move_file("mv file1.txt first_dir/second_dir/third_dir/")`  is a valid command. Your program should move file when destination is a directory as well